tests/reg_issue5268/reg_issue5268.fz.expected_err

```diff
@@ -1,21 +1,66 @@
 
---CURDIR--/reg_issue5268.fz:37:18: error 1: Could not find called feature
+--CURDIR--/reg_issue5268.fz:45:3: error 1: Unused, non public field 'v'
+  v  (i32)->bool := %%2                   # should work
+--^
+To solve this, do either of the following
+  - use the field
+  - set it to 'public'
+  - explicitly ignore the result by using '_' instead of 'v'
+
+
+--CURDIR--/reg_issue5268.fz:37:3: error 2: Unused, non public field 'r'
+  r  ()->bool := *2                       # 3. should flag an error: 0 actual args, expected 1
+--^
+To solve this, do either of the following
+  - use the field
+  - set it to 'public'
+  - explicitly ignore the result by using '_' instead of 'r'
+
+
+--CURDIR--/reg_issue5268.fz:41:3: error 3: Unused, non public field 't'
+  t  (unit, f64, String)->bool := *2      # 7. should flag an error: 3 actual args, expected 1
+--^
+To solve this, do either of the following
+  - use the field
+  - set it to 'public'
+  - explicitly ignore the result by using '_' instead of 't'
+
+
+--CURDIR--/reg_issue5268.fz:50:3: error 4: Unused, non public field 'x'
+  x  ()->i32 := !0x5555_aaaa              # should work
+--^
+To solve this, do either of the following
+  - use the field
+  - set it to 'public'
+  - explicitly ignore the result by using '_' instead of 'x'
+
+
+--CURDIR--/reg_issue5268.fz:55:3: error 5: Unused, non public field 'z'
+  z  (unit,f64,String)->i32 := !?!4711    # should work
+--^
+To solve this, do either of the following
+  - use the field
+  - set it to 'public'
+  - explicitly ignore the result by using '_' instead of 'z'
+
+
+--CURDIR--/reg_issue5268.fz:37:18: error 6: Could not find called feature
```

Comment:
Pleaser add some calls like `ignore v` to this test such that we can see that the cases that should work actually do and those that should flag an error do so as well?

Reply:
I think I can/should not do that since in the test:
```
  # the test cases are all declared twice with only the second version being
  # actually called below.
  #
  # The reason is that the fact that there was an actual call had an effect on the
  # errors that were produced before #5268 was fixed.
```